modules/simple_forms_api/docs/rfcs/0003-form-submission-purge-plan.md

```diff
@@ -42,8 +46,19 @@ The `FormSubmission` table contains user-submitted data, including PII and PHI.
     2. Purge sensitive data from identified records.
     3. Delete associated PDFs from S3.
 
-- **Delay Between Marking and Deleting**:
-  - Optionally mark records for deletion and delay the actual purge to allow recovery if necessary.
+- **ActiveSupport Notification for Deletion**:
+  - Emit an `ActiveSupport::Notification` event when deleting PII:
+    ```ruby
+    ActiveSupport::Notifications.instrument(
```

Comment:
Since this is synchronous, as long as you call this before the actual data purge (perhaps a `pii.deleting` event?), subscribers could access the data needed before deletion and then return back to your job which would finish the process.

Reply:
Good idea! I've updated this area of the document with your idea.